hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/CryptoCreateTransitionLogic.java

```diff
@@ -110,14 +103,8 @@ public void doStateTransition() {
             txnCtx.setStatus(SUCCESS);
 
             final List<ByteString> aliasesToLink = new ArrayList<>();
-            if (!op.getAlias().isEmpty()) {
-                if (op.getAlias().size() == EVM_ADDRESS_SIZE) {
-                    aliasesToLink.add(op.getAlias());
-                } else {
-                    aliasesToLink.add(op.getAlias());
-                    final var key = asPrimitiveKeyUnchecked(op.getAlias());
-                    maybeLinkEvmAddressFrom(key, aliasesToLink);
-                }
+            if (!op.getAlias().isEmpty() && op.getAlias().size() == EVM_ADDRESS_SIZE) {
+                aliasesToLink.add(op.getAlias());
```

Comment:
We do not need to check `op.getAlias().size() == EVM_ADDRESS_SIZE`, right? Since the semantic checks would have already rejected a txn with a non evm address alias

Reply:
Yes, you're right, I will remove it.